rosidl_adapter/rosidl_adapter/parser.py

```diff
@@ -448,36 +448,39 @@ def parse_message_file(pkg_name, interface_filename):
             pkg_name, msg_name, h.read())
 
 
+def extract_file_level_comments(message_string):
+    lines = message_string.splitlines()
+    index = next(
+        (i for i, v in enumerate(lines) if not v.startswith(COMMENT_DELIMITER)), -1)
+    if index != -1:
+        file_level_comments = lines[0:index]
```

Comment:
@ivanpauno nit^2: 
```suggestion
        file_level_comments = lines[:index]
```

Reply:
Done in https://github.com/ros2/rosidl/pull/632/commits/b9710b6620fb8abc4435fc03f2599ce9b9b6fc42